### PR TITLE
feat: Ability to tag just EKS cluster

### PR DESCRIPTION
**Pull request by mmerrill3**
Signed-off-by: mmerril8 <mmerril8@its.jnj.com>

# PR o'clock

## Description

Update to provide ability to tag just EKS resources (#1470 )

### Checklist

- [x] README.md has been updated after any changes to variables and outputs. See https://github.com/terraform-aws-modules/terraform-aws-eks/#doc-generation

### CODE REVIEW

File: cluster.tf [hunk 1]
```diff
@@ -12,7 +12,10 @@ resource "aws_eks_cluster" "this" {
   enabled_cluster_log_types = var.cluster_enabled_log_types
   role_arn                  = local.cluster_iam_role_arn
   version                   = var.cluster_version
-  tags                      = var.tags
+  tags                      = merge(
+    var.cluster_tags,
```
Comment (daroga0002) [inline cluster.tf:16]: I think better will be using:
```
tags = merge(
        var.tags,
        var.cluster_tags)
```

as then you will be able to overwrite some common tags by cluster tags.

As this is how [merge](https://www.terraform.io/docs/language/functions/merge.html) is working

Comment (mmerrill3) [reply cluster.tf:16]: thanks for the feedback, and pointing out how merge's order of precedence works.  I'll make the update and ping you when I'm done.

**Comment by mmerrill3** [comment]
@daroga0002 thanks for the feedback.  I made the suggested edit and rebased.

**Review by daroga0002 — APPROVED** [review]
LGTM

**Review by antonbabenko — APPROVED** [review]

**Pull request closed without merging**